README.md

```diff
@@ -1,22 +1,29 @@
-# Lista de Dados Públicos e Estratégias para Coleta em Massa
+# Serenata de amor
+> Combater a corrupção de forma massiva e eficaz.
 
+*:uk: [English version](README-en.md)*
```

Comment:
What about adding “Jump to the English version”? When I saw this line thought it was the title of the following section (not a shortcut to the next one).